vmm/vmm.go

```diff
@@ -0,0 +1,119 @@
+package vmm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"sync"
+
+	"github.com/bobuhiro11/gokvm/flag"
+	"github.com/bobuhiro11/gokvm/machine"
+	"github.com/bobuhiro11/gokvm/term"
+)
+
+type BootConfig struct {
+	KernFile   *os.File
+	InitrdFile *os.File
```

Comment:
Since these two members are only used for loading in the vmm.Setup() function, how about making them local variables of vmm.Setup()?

Reply:
done